src/commands/clan/done.ts

```diff
@@ -1,6 +1,6 @@
 import { ChatInputCommandInteraction, SlashCommandBuilder } from "discord.js";
 
-import { pool } from "../../db";
+import { databaseClient } from "../../clients/db";
```

Comment:
this is exceedingly minor but is there a way we can move databaseClient from db to database? "db" means disco bandit in a few other places of code and it might be good to be more explicit here.

Reply:
yeah good point